site/_shortcodes/Playlist.js

```diff
@@ -0,0 +1,210 @@
+const {html} = require('common-tags');
+const fetch = require('node-fetch/lib/index');
```

Comment:
You added axios as a dependency but use node-fetch here (which is good), so axios can probably be removed?